gettext_finder.py

```diff
@@ -66,7 +66,7 @@ def start():
     filenames = find_all_files(".", [".py"], IGNORED_DIRS)
     filenames_html = []
     filenames_html += find_all_files('templates', [".html"], IGNORED_DIRS)
-    filenames_js = find_all_files(".", [".js"], IGNORED_DIRS)
+    filenames_js = find_all_files("./build/static/", [".js"], IGNORED_DIRS)
```

Comment:
There are too many JS files in the source directory to parse. We can just parse the built bundle, which includes all our dependencies, but that may actually prove useful at some point.